types/faker/index.d.ts

```diff
@@ -266,13 +266,13 @@ declare namespace Faker {
         };
 
         system: {
-            fileName(ext?: string, type?: string): string;
-            commonFileName(ext: string, type?: string): string;
+            fileName(): string;
+            commonFileName(ext?: string): string;
             mimeType(): string;
             commonFileType(): string;
             commonFileExt(): string;
             fileType(): string;
-            fileExt(mimeType: string): string;
+            fileExt(mimeType?: string): string;
```

Comment:
Made it optional, see here:
https://github.com/Marak/faker.js/blob/c9764abd20a198e318367952a94587eda1bac645/lib/system.js#L172

Reply:
Please add new test.
```typescript
resultStr = faker.system.fileExt();

```

Reply:
Please add new optional test

```typescript
resultStr = faker.system.fileExt();
```